docs/code/MlNetCookBook.md

```diff
@@ -578,6 +578,60 @@ var biases = modelParameters.GetBiases();
 
 ```
 
+## How do I get a model's weights to look at the global feature importance?
+The below snippet shows how to get a model's weights to help determine the feature importance of the model for a linear model.
+
+```csharp
+var linearModel = model.LastTransformer.Model;
+
+var weights = linearModel.Weights;
+```
+
+The below snipper shows how to get the weights for a fast tree model.
+
+```csharp
+var treeModel = model.LastTransformer.Model;
+
+var weights = new VBuffer<float>();
+treeModel.GetFeatureWeights(ref weights);
+```
+
+## How do I look at the global feature importance?
+The below snippet shows how to get a glimpse of the the feature importance, or how much each feature impacts the performance of the model. It also outputs the difference in root mean squared for each feature as though the feature were replaced with a random value.
```

Comment:
>It also outputs the difference in root mean squared for each feature as though the feature were replaced with a random value. [](start = 139, length = 125)

"Permutation Feature Importance works by computing the change in the evaluation metrics when each feature is replaced by a random value. In this case, we are investigating the change in the root mean squared error".